install.sh

```diff
@@ -121,42 +123,96 @@ setup_ramalama() {
   local branch="${BRANCH:-s}"
   local url="${host}/containers/ramalama/${branch}/bin/${from_file}"
   local to_file="$TMP/${from_file}"
+  local sysdir
+  get_sysdir
+
   if $local_install; then
     url="bin/${from_file}"
   fi
 
   download "$url" "$to_file"
-  local ramalama_bin="${1}/${binfile}"
-  local syspath
-  syspath=$(get_installation_dir)
+  local max_jobs
```

Comment:
Optimisation, we knock a couple of 100 milliseconds off install time by pulling multiple files in parallel